samples/bluetooth/peripheral_mds/sysbuild/hci_rpmsg/prj.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_IPC_SERVICE=y
+CONFIG_MBOX=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_HCI_RAW_RESERVE=1
+#CONFIG_BT_MAX_CONN=16
```

Comment:
Why is it commented here?

Reply:
I think this line can be removed or set to desired value 2 instead of having line 30

Reply:
It is commented as that part of the file is the copy from the upstream file, they can probably be merged better at some point